src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -2681,13 +2687,61 @@ static WellKnownMembers()
                 0,                                                                                                          // Arity
                     (byte)SignatureTypeCode.GenericTypeParameter, 6,                                                        // Field Signature
 
+                // System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5_T6_T7_TRest__Item1
+                (byte)MemberFlags.Field,                                                                                    // Flags
+                (byte)WellKnownType.ExtendedType, WellKnownType.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5_T6_T7_TRest.ExtensionByte(), // DeclaringTypeId
```

Comment:
Does ```value.ExtensionByte()``` need to be a compile-time constant so the ```byte[]``` is stored as raw bytes in the assembly?